manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JTL.Wawi
+PackageVersion: 1.5.47.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
```

Comment:
You do know the InstallerLocale is actually de-DE right? 🤔😁

Reply:
Yeah i am going to fix it in the next Version